Gruntfile.js

```diff
@@ -109,7 +109,22 @@ module.exports = function(grunt) {
 			options: {
 
 			}
-		}
+		},
+
+		'jshint-jsx': {
+			options: {
```

Comment:
Out of curiosity, why do you specify all of these optiions inline, rather than in a `.jshintrc` file in the root?  Some editors will do autolinting as you write if they find a `.jshintrc` file at the root.